### DISCUSSION TITLE
The management interface ui cannot be opened on the normal page. Many versions are like this. You can only enter and open normally on the incognito page of Google Chrome.

### DISCUSSION THREAD

**Discussion by ghost** [rabbitmq/rabbitmq-server#3617] [Other] [answered]
![image](https://user-images.githubusercontent.com/71524776/139141043-90812b19-18a3-433e-87a5-95643d1bda94.png)

![image](https://user-images.githubusercontent.com/71524776/139140391-4142302a-0481-454e-bcc1-5ecedf16c896.png)

![image](https://user-images.githubusercontent.com/71524776/139140602-8d971533-6f17-4c43-b281-6e9b81a9163f.png)

I hope I can help with the investigation, which greatly affects the use.thanks~

**Comment by ghost** [comment]
Oh, and Firefox also opened the privacy window and went straight in

![image](https://user-images.githubusercontent.com/71524776/139141525-b9659816-cdb9-438a-b26a-a0fc7b8fee32.png)

**Comment by mkuratczyk** [comment]
This is a browser cache issue. Similar report from yesterday: https://github.com/rabbitmq/rabbitmq-server/discussions/3612. A hard refresh of the page should fix this. I guess there is a too long expiration time set on some element(s) - if you can identify what part of that page is cached for a long time, that could help fix the problem perhaps. But the workaround is simple.

**Comment by ghost** [comment]
Thank you for your reply, but this problem always exists. It is a problem when a system logs in normally first, which greatly affects the experience. Can you optimize this, including the blank problem after entering

**Comment by ghost** [comment]
You can quickly start a test with local docker to see, and the same is true for completely clearing the cache
![image](https://user-images.githubusercontent.com/71524776/139149622-c677c3b2-95ac-40e6-b451-7df67bc0385a.png)

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment, marked as answer]
As @mkuratczyk suggested above, this is a browser caching issue. Clear the cache after upgrading as the docs suggest. There is only so much that can be changed in the current management UI, we plan to develop another one at some point. The UI code also doesn't change particularly often.

**Reply by ghost** [reply]
Ok thank you very much
